symengine/prime_sieve.cpp

```diff
@@ -1,6 +1,8 @@
 #include <symengine/prime_sieve.h>
+#include <ciso646>
```

Comment:
Perhaps this should only be included conditionally. It seems to have been removed in C++20: https://en.cppreference.com/w/cpp/header/ciso646

Reply:
We already include it unconditionally in symengine_rcp.h

Reply:
I guess it would be ok as long as no compilation fails because of it.